Misc/NEWS.d/next/Library/2018-03-29-04-32-25.bpo-33175._zs1yM.rst

```diff
@@ -0,0 +1,2 @@
+In dataclasses, Field.__set_name__ now looks up the __set_name__ special
```

Comment:
Does this need a NEWS entry if there was no release with the buggy code? (I'm mostly curious; you'll know the norms better than I do.)

Reply:
I thought about this, but unfortunately I think it does need the NEWS blurb. Ned has already cut 3.7.0b3, so the bug will be in that release. Thanks again for catching this.